src/arns.lua

```diff
@@ -527,7 +527,9 @@ end
 --- @param demandFactor number The demand factor
 --- @return number permabuyFee - the permabuy fee
 function arns.calculatePermabuyFee(baseFee, demandFactor)
-	local permabuyPrice = baseFee + arns.calculateAnnualRenewalFee(baseFee, constants.PERMABUY_LEASE_FEE_LENGTH)
+	local permabuyPrice = baseFee + arns.calculateAnnualRenewalFee(baseFee, constants.PERMABUY_LEASE_FEE_LENGTH_YEARS)
+	print("permabuyPrice: " .. permabuyPrice)
+	print("demandFactor: " .. demandFactor)
```

Comment:
intended?

Reply:
removed in [0ef9f20](https://github.com/ar-io/ar-io-network-process/pull/378/commits/0ef9f204492c5250493154208cfe347d606f171b)